src/ui/zcl_abapgit_popups.clas.abap

```diff
@@ -100,7 +100,7 @@ ENDCLASS.
 
 
 
-CLASS zcl_abapgit_popups IMPLEMENTATION.
+CLASS ZCL_ABAPGIT_POPUPS IMPLEMENTATION.
```

Comment:
What about to stop using pretty printer and just keep everything lowercase until uppercase required?

Reply:
activation of the class makes it upper case, its a built-in "feature"

Reply:
my ABAP source codes are all lowercase, the previous version is lowercase (abapGit versions only activated code), so when did they remove the mentioned uppercasing-at-activation feature?

Reply:
its editor dependent, I would like not to force developers to choose a specific editor

Reply:
Nobody wants to force anybody to use 1 editor. But would be nice if our editors didn't do any unnecessary formatting changes.

Reply:
BTW abapGit has the nice feature called Patch where you can single out only the changes you really want to commit.